variants/mkrnb1500/variant.cpp

```diff
@@ -254,7 +254,7 @@ void initVariant() {
 
   // put SARA modem in reset on start to conserve power if it's not used
   pinMode(SARA_RESETN, OUTPUT);
-  digitalWrite(SARA_RESETN, HIGH);
+  digitalWrite(SARA_RESETN, LOW);
```

Comment:
Hi @giulcioffi :coffee: So we are still putting the SARA in reset here. Is that still necessary given the changes in https://github.com/arduino-libraries/MKRNB/pull/55 ? Cheers, Alex - Otherwise great PR ;) let's hash this one out in Slack and then we should be able to merge it in.

Reply:
All cleared ;)